src/View/Contact/contact.html.twig

```diff
@@ -32,10 +32,104 @@
 
                 </section>          
             </div>
- {% include 'components/contact_form.html.twig' %}
+<h2 class="center">Contactez-nous</h1>    
```

Comment:
h1 instead of h2

Reply:
it's h2 tag, i have a h1 in the beginning (<h1 class="center">Oxygen School : Qui sommes-nous</h1>)